### ISSUE TITLE
missing dependencies for humble

### ISSUE THREAD

**Issue by tylerjw** [MetroRobots/ros_system_fingerprint#5]
Here are the commands I ran to install and setup my enviroment:

```
sudo apt-get install ros-humble-system-fingerprint
source /opt/ros/humble/setup.bash
```

When I run the command in the readme I get this error:
```
rosrun system_fingerprint imprint

Command 'rosrun' not found, but can be installed with:
sudo apt install rosbash
```

If I instead try to run this with ros2 tooling I get this python error:
```
ros2 run system_fingerprint imprint
Traceback (most recent call last):
  File "/opt/ros/humble/lib/system_fingerprint/scripts/imprint", line 33, in <module>
    sys.exit(load_entry_point('UNKNOWN==0.5.0', 'console_scripts', 'imprint')())
  File "/opt/ros/humble/lib/system_fingerprint/scripts/imprint", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/ros/humble/lib/python3.10/site-packages/system_fingerprint/__init__.py", line 6, in <module>
    from ros2action.api import get_action_names_and_types
ModuleNotFoundError: No module named 'ros2action'
[ros2run]: Process exited with failure 1
```

**Comment by tylerjw** [comment]
I am on 22.04 and if I try to follow the instructions to install rosbash I get this error:

```
sudo apt install rosbash
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python3-catkin-pkg : Conflicts: catkin but 0.8.10-7 is to be installed
 python3-catkin-pkg-modules : Conflicts: catkin but 0.8.10-7 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by tylerjw** [comment]
If I go to rosindex to find rosbash to see if I should install it some otherway I see that it is part of ros1:

https://index.ros.org/p/rosbash/github-ros-ros/#noetic

**Comment by tylerjw** [comment]
I tried manually installing ros-humble-ros2action, but now when I run it I get an error about missing ros_system_fingerprint. But I can't figure out how to install that module:

```
ros2 run system_fingerprint imprint
Traceback (most recent call last):
  File "/opt/ros/humble/lib/system_fingerprint/scripts/imprint", line 33, in <module>
    sys.exit(load_entry_point('UNKNOWN==0.5.0', 'console_scripts', 'imprint')())
  File "/opt/ros/humble/lib/system_fingerprint/scripts/imprint", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/ros/humble/lib/python3.10/site-packages/system_fingerprint/imprint.py", line 5, in <module>
    from ros_system_fingerprint import modules
ModuleNotFoundError: No module named 'ros_system_fingerprint'
[ros2run]: Process exited with failure 1
```

**Comment by tylerjw** [comment]
I looked on ros-index and that package doesn't exist: https://index.ros.org/p/system_fingerprint/github-MetroRobots-ros_system_fingerprint/#humble

But maybe it is a module inside this python pacakge?

**Comment by DLu** [comment]
So the command `rosrun system_fingerprint imprint` comes from the `noetic` branch (the default on GitHub). If you switch to the `ros2` branch, it says `ros2 run system_fingerprint imprint`. So the `rosbash` stuff is not relevant. 

The other problems are now addressed in #7. Thanks for testing things out.
